src/server/scripts/EasternKingdoms/Karazhan/instance_karazhan.cpp

```diff
@@ -257,6 +258,7 @@ class instance_karazhan : public InstanceMapScript
                             break;
                         case DONE:
                             HandleGameObject(m_uiGamesmansExitDoor, true);
+                            instance->ToInstanceMap()->PermBindAllPlayers();
```

Comment:
Shouldn't the binding of players happen on the first pull of a boss, or is it handled differently in the chess encounter?

Reply:
They are bound when boss dies, and the chess has no boss since its an event

Reply:
Let me rephrase. I always thought you got bound at the start of an encounter. But maybe that is just retail behaviour

Reply:
> Let me rephrase. I always thought you got bound at the start of an encounter. But maybe that is just retail behaviour

On AC at least it's always after the first boss dies. I think it was like this in wotlk as well